### PR TITLE
[NEW] Autocomplete emojis

### LINKED ISSUE #178: [NEW] Show emotions in the autocomplete view

### PR DESCRIPTION
**Pull request by cardoso**
@RocketChat/ios

|Preview|
|-|
||

Closes #178

### CODE REVIEW

File: Rocket.Chat/Controllers/Chat/ChatControllerAutocomplete.swift [hunk 1]
```diff
@@ -80,6 +93,21 @@ extension ChatViewController {
         if let user = searchResult[indexPath.row].1 as? User {
             cell.avatarView.labelInitials.textColor = .white
             cell.avatarView.user = user
+        } else if let emoji = searchResult[indexPath.row].1 as? Emoji {
+            if let cell = autoCompletionView.dequeueReusableCell(withIdentifier: "EmojiAutocompleteCell") as? EmojiAutocompleteCell {
```
Comment (rafaelks) [inline Rocket.Chat/Controllers/Chat/ChatControllerAutocomplete.swift]: `EmojiAutocompleteCell` could be an static identifier.

File: Rocket.Chat/External/RCEmojiKit/EmojiSearcher.swift [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            return nil
+        }.sorted {
+            ($0.suggestion.count - shortname.count) < ($1.suggestion.count - shortname.count)
+        }
+    }
+}
+
+public extension EmojiSearcher {
+    public static let standard = EmojiSearcher(emojis:
+            Emojione.people +
```
Comment (rafaelks) [inline Rocket.Chat/External/RCEmojiKit/EmojiSearcher.swift]: What do you thin about having a `Emojione.all` that returns all categories?

File: Rocket.Chat/External/RCEmojiKit/Views/EmojiAutocompleteCell.swift [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    func highlight(string: String) {
+        if let attributedString = shortnameLabel.attributedText {
+            let attributedString = NSMutableAttributedString(string: attributedString.string)
+
+            if let range = attributedString.string.range(of: string) {
+                attributedString.addAttribute(NSAttributedStringKey.foregroundColor, value: UIColor.black, range: NSRange(range, in: attributedString.string))
+
+                shortnameLabel.attributedText = attributedString
+            }
+        }
```
Comment (rafaelks) [inline Rocket.Chat/External/RCEmojiKit/Views/EmojiAutocompleteCell.swift]: 👍

**Comment by cardoso** [comment]
Thanks @rafaelks! I made the suggested improvements.

**Pull request merged**